presto-main/src/main/java/com/facebook/presto/cost/JoinStatsRule.java

```diff
@@ -49,12 +49,21 @@
         implements ComposableStatsCalculator.Rule
 {
     private static final Pattern PATTERN = Pattern.typeOf(JoinNode.class);
+    private static final double DEFAULT_UNMATCHED_ANTI_JOIN_NDVS_COEFFICIENT = 0.5;
```

Comment:
What value would make most sense with our otherwise assumption of uniform distribution?

q05, q40, q49, q72, q75, q77, q78, q80, and q93 are the queries in TPC-DS that have LEFT JOINs, we should benchmark those in particular with and without your change